app/views/hyrax/homepage/_featured_researcher.html.erb

```diff
@@ -1,7 +1,5 @@
-<%= editable_content_block @featured_researcher, true %>
-
-<ul class="list-inline collection-highlights-list">
```

Comment:
I have temporarily removed this capability (of storing and displaying multiple featured researchers). I'm going to come back to this next.